lib/rubocop/cop/packaging/require_hardcoding_lib.rb

```diff
@@ -65,7 +64,7 @@ def on_new_investigation
         # More about the `#on_send` method can be found here:
         # https://github.com/rubocop-hq/rubocop-ast/blob/08d0f49a47af1e9a30a6d8f67533ba793c843d67/lib/rubocop/ast/traversal.rb#L112
         def on_send(node)
-          return unless require?(node)
+          return unless File?(node)
```

Comment:
We need a different name here, `File` is not very clear as to what it implies or means. Also preferably something that's not capitalized. Hope that makes sense?

Reply:
Yes, of course, thanks for PTO. :)